BUILDIT.md

```diff
@@ -5776,11 +5799,323 @@ we expect the persisted value to be
 one hour *less* than what the person inputted. 
 
 
-# 16. Run the _Finished_ MVP App!
+# 16. Reordering `items` Using Drag & Drop
+
+At present `people` using the `App`
+can only add new `items` to a stack
+where the newest is on top; no ordering.
+
+People who tested the `MVP`
+noted that the ability to **reorder `items`**
+was an **_essential_ feature**:
+[dwyl/mvp/**issues/145**](https://github.com/dwyl/mvp/issues/145)
+
+So in this step we are going to 
+add the ability to organize `items`.
+We will implement reordering using 
+**drag and drop**!
+And by using `Phoenix Liveview`,
+**other people** will also be able 
+to **see the changes in real time**!
+
+
+
+
+##  16.1 `Item` schema changes
+
+By introducing this feature 
+(and so everyone sees the correct positioning of each item),
+we ought to add a new field:
+**`position`**.
+This new field called **`position`** will be an `integer`,
+referencing the *index of the item within the list*.
+
+The `position` field can't be under `0` 
+and will dynamically change according to the position of the item in the list.
+
+With this in mind,
+let's add this field in our migration 
+and schema definition files.
+
+Open `priv/repo/migrations/20220627162154_create_items.exs`
```

Comment:
I have removed this section and reverted this change to the migration. ✂️ 
But in future, please avoid editing previous migrations as this strategy does not work in production. 👍